router/throttler/factory_test.go

```diff
@@ -103,6 +103,16 @@ func TestFactory(t *testing.T) {
 			"destType":      "destName",
 		}).LastValue())
 	})
+
+	t.Run("when no limit is set", func(t *testing.T) {
+		config := config.New()
+		config.Set("Router.throttler.adaptive.enabled", true)
```

Comment:
[Optional]: Variable 'config' collides with imported package name 
```suggestion
		conf := config.New()
		conf.Set("Router.throttler.adaptive.enabled", true)
```